server.go

```diff
@@ -971,12 +992,14 @@ func (tb *internalTraceBackend) Close() error {
 
 // SendSync sends the span directly into the veneur Server.
 func (tb *internalTraceBackend) SendSync(ctx context.Context, span *ssf.SSFSpan) error {
-	if tb.spanWorker == nil {
+	if tb.spanWorkers == nil {
 		return ErrNoSpanWorker
 	}
 	ch := make(chan struct{})
+
+	workerID := span.Id % int64(len(tb.spanWorkers))
```

Comment:
I think this warrants a comment that while some span sinks might not care about per-traceid grouping, others *do*. Datadog, for example, like spans grouped by trace id. Future spelunkers may wonder the same thing?

Reply:
OK, I guess from the performance I've observed, we should probably investigate not splitting up the channels (and feeding the workers from a single channel), then (also it should probably use the `TraceId` and not the span ID). The DD trace sink can still re-group them (there's only one of it anyway).